modules/abstract-eth/src/lib/transferBuilder.ts

```diff
@@ -93,8 +95,9 @@ export class TransferBuilder {
     throw new InvalidParameterValueError('Invalid expiration time');
   }
 
-  signAndBuild(chainId?: string): string {
+  signAndBuild(chainId?: string, coinUsesNonPackedEncodingForTxData?: boolean): string {
     this._chainId = chainId;
+    this._coinUsesNonPackedEncodingForTxData = coinUsesNonPackedEncodingForTxData;
```

Comment:
I'm not sure why are we initializing the variable here, which is not used in the function itself?
Probably better to have a separate function to set like `expirationTime` 
or you could initialize in the constructor ?

Reply:
The reason why we did it like this is so that there are no changes required wherever transferBuilder is being called. If it is done as a separate method like expirationTime, then we will have to modify the places in SDK/Wallet platform, and it will be a breaking change. The reason why it is initialized in this method is because signAndBuild is being called from transactionBuilder as shown [here](https://github.com/BitGo/BitGoJS/blob/fb84e7ffafaf655b87b71c5019df910760835499/modules/abstract-eth/src/lib/transactionBuilder.ts#L617) and hence we used this method to initialize this param which is going to be used in during building the transaction